docs/source/_toctree.yml

```diff
@@ -0,0 +1,6 @@
+- sections:
+  - local: index
+    title: 🤗 Optimum Habana
+  - local: quickstart
+    title: Quickstart
+  title: 🤗 Optimum Habana
```

Comment:
I added a title so it renders in the table of contents - happy to change this if needed!